test/test-server.js

```diff
@@ -1,25 +1,25 @@
 var assert = require('assert');
-var fs     = require('fs');
-var hl7    = require('../lib/index.js');
-var net    = require('net');
-var path   = require('path');
+var fs = require('fs');
+var hl7 = require('../lib/index.js');
+var net = require('net');
+var path = require('path');
 var server = hl7.Server;
 
-
 var VT = String.fromCharCode(0x0b);
 var FS = String.fromCharCode(0x1c);
 var CR = String.fromCharCode(0x0d);
 
-describe('FileServer', function() {
+describe('FileServer', function () {
```

Comment:
Running these tests depend on running all tests. This is problematic, but it works for now.